From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Shelley Vohr <shelley.vohr@gmail.com>
Date: Mon, 20 Jul 2020 19:27:57 -0700
Subject: Revert swiftshader roll in DEPS

https://chromium-review.googlesource.com/c/chromium/src/+/2304381 and
https://chromium-review.googlesource.com/c/chromium/src/+/2306053
rolled swiftshader to ea6f21a052c9 and caused a breakage in arm64 jobs
on macOS:

Undefined symbols for architecture arm64:
  "llvm::sys::RWMutexImpl::lock_shared()", referenced from:
      llvm::sys::SmartRWMutex<true>::lock_shared() in PassRegistry.o
  "llvm::sys::RWMutexImpl::unlock_shared()", referenced from:
      llvm::sys::SmartRWMutex<true>::unlock_shared() in PassRegistry.o
  "llvm::sys::RWMutexImpl::lock()", referenced from:
      llvm::sys::SmartRWMutex<true>::lock() in PassRegistry.o
  "llvm::sys::RWMutexImpl::unlock()", referenced from:
      llvm::sys::SmartRWMutex<true>::unlock() in PassRegistry.o
  "llvm::sys::RWMutexImpl::RWMutexImpl()", referenced from:
      llvm::object_creator<llvm::PassRegistry>::call() in PassRegistry.o
  "llvm::sys::RWMutexImpl::~RWMutexImpl()", referenced from:
      llvm::PassRegistry::~PassRegistry() in PassRegistry.o
ld: symbol(s) not found for architecture arm64

Tracking bug at: crbug.com/1107706

diff --git a/DEPS b/DEPS
index bb6d0251a8c42c60dafdc39632aed21104b86545..881c014addcba1a1e36d69bf8d3637a487b0fe21 100644
--- a/DEPS
+++ b/DEPS
@@ -211,7 +211,7 @@ vars = {
   # Three lines of non-changing comments so that
   # the commit queue can handle CLs rolling SwiftShader
   # and whatever else without interference from each other.
-  'swiftshader_revision': 'f272427bda39a1abf3aa6e1a1bdd9b628b5ea5b1',
+  'swiftshader_revision': 'e8dd233c7a85f3c689caf06c226a7f8405a480d3',
   # Three lines of non-changing comments so that
   # the commit queue can handle CLs rolling PDFium
   # and whatever else without interference from each other.
